docs/gotchas.rst

```diff
@@ -0,0 +1,140 @@
+.. _gotchas:
+
+Gotchas/FAQ
+===========
+
+This is a page for some suggestions, gotchas, and FAQs.
+
+Also see: :ref:`examples`
+
+
+`+init=<auth>:<auth_code>` should be replaced with `<auth>:<auth_code>`
+-----------------------------------------------------------------------
+
+The `+init=<auth>:<auth_code>` syntax is deprecated and will be removed
+in future versions of PROJ. Also, if you use the `+init` syntax,
+you may have problems initializing projections whe the other syntax works.
+
+.. code-block:: python
+
+    >>> from pyproj import CRS
+    >>> CRS("ESRI:54009") 
+    <Projected CRS: ESRI:54009>
+    Name: World_Mollweide
+    Axis Info [cartesian]:
+    - E[east]: Easting (metre)
+    - N[north]: Northing (metre)
+    Area of Use:
+    - name: World
+    - bounds: (-180.0, -90.0, 180.0, 90.0)
+    Coordinate Operation:
+    - name: World_Mollweide
+    - method: Mollweide
+    Datum: World Geodetic System 1984
+    - Ellipsoid: WGS 84
+    - Prime Meridian: Greenwich
+
+    >>> CRS("+init=ESRI:54009") 
+    ...
+    pyproj.exceptions.CRSError: Invalid projection: +init=ESRI:54009 +type=crs: (Internal Proj Error: proj_create: cannot expand +init=ESRI:54009 +type=crs)
+
+
+Proj (Not a generic latitude/longitude to projection converter)
+---------------------------------------------------------------
+
+:class:`~pyproj.proj.Proj` is limited is limited to converting between geographic and 
```

Comment:
twice "is limited"

Reply:
Thanks! I updated it later. So, the change is in master now.